src/helper/sequelize_connection_helper.ts

```diff
@@ -0,0 +1,67 @@
+import { Answer } from "../database/models/Answer"
+import { Question } from "../database/models/Question"
+import { QuestionDAO } from "../database/dao/question_dao"
+import {
+	QuestionPersistentModel,
+	AnswerPersistentModel
+} from "../model/persistent/persistent_type"
+import { AnswerDAO } from "../database/dao/answer_dao"
+import { Sequelize } from "../../node_modules/@types/sequelize"
+import { DatabaseConfigurator } from "../database/database_configurator"
+
+// Open Connection Helper
+const OpenConnectionHelper = async (
```

Comment:
we might change that function name to `OpenConnection` ? what do you think ? the word helper there is not adding additional information anyway